geometric/tests/test_run_json.py

```diff
@@ -0,0 +1,56 @@
+"""
+A set of tests for using the QCEngine project
+"""
+
+import numpy as np
+import json, os, shutil
+from . import addons
+import geometric
+
+@addons.using_qcengine
+@addons.using_rdkit
+def test_run_json():
+    # create a test input json file
+    qc_schema_input = {
+        "schema_name": "UFF gradient",
```

Comment:
The `schema_name` here is `qc_schema_input`.

Reply:
Done